src/Pages/Music.elm

```diff
@@ -62,7 +62,7 @@ init shared url =
     ( { currentlyPlaying = shared.currentlyPlaying, album_list = shared.album_list, route = url.route, song_list = shared.song_list }
     , sendActions
         [ """{"jsonrpc": "2.0", "method": "AudioLibrary.GetSongs", "params": { "properties": [ "artist", "duration", "album", "track", "genre", "albumid","thumbnail" ], "sort": { "order": "ascending", "method": "track", "ignorearticle": true } }, "id": "libSongs"}"""
-        , """{"jsonrpc": "2.0", "method": "AudioLibrary.GetAlbums", "params": { "properties": ["playcount", "artist", "genre", "rating", "thumbnail", "year", "mood", "style", "dateadded"] }, "id": "libAlbums"}"""
+        , """{"jsonrpc": "2.0", "method": "AudioLibrary.GetAlbums", "params": { "properties": ["playcount", "artist", "genre", "rating", "thumbnail", "year", "mood", "style", "dateadded","fanart"] }, "id": "libAlbums"}"""
```

Comment:
You seem to be missing a space before fanart. Doesn't really matter, but I like to keep it tidy :)